docs/Researcher/user-interface/trainings.md

```diff
@@ -28,23 +28,24 @@ To add a training:
-7. Open the *Volume* pane, and press *Volume* to add a volume to your training.
+3. In the *Multi-node* pane, choose `Single node` for a single node training, or `Multi-node (distributed)` for distributed training. When you choose `Multi-node`, you will then choose a framework that is listed. Then choose the `multi-node` training configuration by selecting either `Workers & master` or `Workers only`.
+4. In the *Templates* pane, select a template from the list. Use the search box to find templates that are not listed. If you can't find the specific template you need, see your system administrator.
+5. In the *Training name* pane, enter a name for the *Traninng*, then press continue.
+6. In the *Environment* pane select or [create a new environment](workspaces/create/create-env.md). Use the search box to find environments that are not listed. Press *More settings* to add an `Environment variable` or to edit the *Command* and *Arguments* field for the environment you selected.
+7. In the *Compute resource* pane, select resources for your tranings or [create a new compute resource](workspaces/create/create-compute.md). Use the search box to find resources that are not listed. Press *More settings* to use **Node Affinity** to limit the resources to a specific node.
+8. Open the *Volume* pane, and press *Volume* to add a volume to your training.
    1. Select the *Storage class* from the dropdown.
    2. Select the *Access mode* from the dropdown.
    3. Enter a claim size, and select the units.
    4. Select a *Volume system*, mode from the dropdown.
    5. Enter the *Container path* for volume target location.
    6. Select a *Volume persistency.
-8. In the *Data sources* pane, press *add a new data source*. For more information, see [Creating a new data source](workspaces/create/create-ds.md) When complete press, *Create Data Source*.
-9. In the *General* pane, add special settings for your training (optional):
+9.  In the *Data sources* pane, press *add a new data source*. For more information, see [Creating a new data source](workspaces/create/create-ds.md) When complete press, *Create Data Source*.
+10. In the *General* pane, add special settings for your training (optional):
    1. Press *Auto-deletion* to delete the training automatically when it either completes or fails. You can configure the timeframe in days, hours, minuets, and seconds. If the timeframe is set to 0, the training will be deleted immediately after it completes or fails.
    2. Press *Annotation* to a name and value to annotate the training. Repeat this step to add multiple annotations.
    3. Press *Label* to a name and value to label the training. Repeat this step to add multiple labels.
-10. When complete, press *Create training*.
+11. When complete, press *Create training*.
 
 ## Managing Trainings
```

Comment:
but where is all the explanation of the muster setup & workers setup?
I don't see any aspect of "multi-node training" on this page that allegedly supposed to depict "how to submit-dist-training"

what am I missing?

Reply:
I will need to add that to the procedure, or break it out into 2 different ones.

Reply:
As far as I see it, I can create a distributed from the very place I submit regular training, same goes for the documentation.

Reply:
@jasonnovichRunAI I am sorry but I need us to meet for 15 minutes to talk this section over I don't fully understand the stricture & I may be missing something.